modules/rostests/apitests/umkm/precomp.h

```diff
@@ -0,0 +1,15 @@
+#ifndef _UMKM_APITEST_PRECOMP_H_
+#define _UMKM_APITEST_PRECOMP_H_
```

Comment:
Question: Is it possible to use #pragma once here instead? I am always unsure regarding that in the precompiled-headers-context.

Reply:
Sure. But if this is something that actually bothers you, you should rather look through our SDK, because those headers are more likely to make any difference.